DB_Proxy.php

```diff
@@ -631,17 +631,17 @@ function processingRequest($options, $access = null, $bypassAuth = false)
         $this->logger->setDebugMessage("[processingRequest]", 2);
 
         $this->outputOfProcessing = array();
-        $generatedPrivateKey = '';
-        $passPhrase = '';
-
+//        $generatedPrivateKey = '';
+//        $passPhrase = '';
+//
         $currentDir = dirname(__FILE__) . DIRECTORY_SEPARATOR;
-        $currentDirParam = $currentDir . 'params.php';
-        $parentDirParam = dirname(dirname(__FILE__)) . DIRECTORY_SEPARATOR . 'params.php';
-        if (file_exists($parentDirParam)) {
-            include($parentDirParam);
-        } else if (file_exists($currentDirParam)) {
-            include($currentDirParam);
-        }
+//        $currentDirParam = $currentDir . 'params.php';
+//        $parentDirParam = dirname(dirname(__FILE__)) . DIRECTORY_SEPARATOR . 'params.php';
+//        if (file_exists($parentDirParam)) {
+//            include($parentDirParam);
+//        } else if (file_exists($currentDirParam)) {
+//            include($currentDirParam);
+//        }
 
```

Comment:
I think any variables come from params.php don't exist in the processingRequest method any more.